entries/e/eqbank.ca.json

```
@@ -0,0 +1,15 @@
+{
+  "EQ Bank": {
+    "domain": "eqbank.ca",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://www.eqbank.ca/about-us/help/common-questions#what-security-features-does-the-eq-bank-card-have",
+    "regions": [
+      "ca"
```

```suggestion
      "ca",
      "fr"
```
Looks like it’s available in France as well :)

---

I suspect it's more that they offer services to Quebec (fr-CA, not fr-FR): https://www.eqbank.ca/about-us/about/company-overview